upgrade/steps.go

```diff
@@ -1021,7 +1025,9 @@ var planPluginSDKUpgrade = stepv2.Func12E("Planning Plugin SDK Upgrade", func(
 	currentBranch, ok := refs.labelOf(currentRef)
 	if !ok {
 		// use latest versioned branch
```

Comment:
Looks like this comment was inaccurate - it would previously return "" which got picked up on https://github.com/pulumi/upgrade-provider/blob/1b9fe00eda2b8a52416099d3903a850a689d4699/upgrade/upgrade_provider.go#L213 to mean don't upgrade.